python-backend/app/api/permits/permit/models/permit.py

```diff
@@ -1,8 +1,5 @@
-from datetime import datetime
```

Comment:
Why is date time no longer being used?

Reply:
I'm not sure when the change was made, but it was just a unused import at this point so i just did cleanup.
Functionally im not sure when/why the usage for it was removed